kubernetes/loculus/values.yaml

```diff
@@ -1742,8 +1745,17 @@ defaultOrganisms:
         configFile:
           <<: *preprocessingConfigFile
           log_level: DEBUG
-          nextclade_dataset_name: community/pathoplexus/cchfv
-          genes: [RdRp, GPC, NP]
+          minimizer_index: https://loculus-project.github.io/nextclade-sort-minimizers/data/CCHF/minimizer.json
```

Comment:
I've forgotten why we're going from `minimizer_url` to `minimizer_index`? It's a URL after all?